.github/workflows/vpdq-ci-cpp.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 defaults:
```

Comment:
(unsure): I remember something about this being weird, where maybe the old CI would run instead of the new one until merge. Since it seems like you are planning on staying with the project a bit longer, we can just make sure it doesn't stop failing afterwards :P